### PR TITLE
Adds basic rbac tests

### PR DESCRIPTION
**Pull request by chuckbutler**
Known deficiencies:
 - Missing test for ingress controller migration. (wip in a future branch)

This will validate that the components have been configured with token based authentication and probes the kubeconfigs for token: lines as well as entries in known_tokens.

### CODE REVIEW

File: tests/20-rbac-test.py [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+        # post rbac component configs
+        for worker in self.workers:
+            kubelet, rc = worker.run('grep token: /root/cdk/kubeletconfig')
+            self.assertTrue(rc == 0)
+            proxy, rc = worker.run('grep token: /root/cdk/kubeproxyconfig')
+            self.assertTrue(rc == 0)
+            self.assertTrue(kubelet != proxy)
+
+    def test_cluster_info(self):
```
Comment (Cynerva) [inline tests/20-rbac-test.py]: A lot of these tests look to be duplicates from 20-charm-validation.py. Mind removing the duplicates?

Comment (chuckbutler) [inline tests/20-rbac-test.py]: not at all, deleted and pushed in a141da0

**Comment by ktsakalozos** [comment]
Do you want this merged on the master or on the enable-rbac branch of juju-solutions ?

**Comment by chuckbutler** [comment]
@ktsakalozos I think it's fine to leave this open until the RBAC code is ready unless a branch is needed to get these moving in CI.

**Comment by ktsakalozos** [comment]
LGTM

**Pull request closed without merging**